src/models/index.ts

```diff
@@ -0,0 +1,8 @@
+import { Organization as OrganizationModel } from "./Organization";
+import { OrganizationUser as OrganizationUserModel } from "./OrganizationUser";
+import { User as UserModel, UserRole as UserRoleEnum } from "./User";
+
+export const OrganizationUser = OrganizationUserModel;
```

Comment:
why we need these aliases????????

Reply:
The aliases allow me to import the models inside the same file and to export them using their original names. This way I am sure that the models are used inside this file, so that when they are imported from it then they are already initialized properly.
In order to avoid this aliases I can export a default object containing all the models, then it will be used in the code whenever any model will be required.

Reply:
the problem here is that models get initialized as a side effect of importing the model file - instead model initialization should be explicit, that's the real issue here

each model definition file should export an `init` method that you'll call on app initialization

by doing that, side effects are only executed once and explicitly - then you can import and use the models multiple times

this `index.ts` file is actually a workaround for the problem caused by calling `init` in the model files